app/views/common/EditView.coffee

```diff
@@ -0,0 +1,71 @@
+require('app/styles/editor/resource/edit.sass')
+RootView = require 'views/core/RootView'
+template = require 'app/templates/editor/common/edit'
+ConfirmModal = require 'views/core/ConfirmModal'
+PatchesView = require 'views/editor/PatchesView'
+errors = require 'core/errors'
+
+require 'lib/game-libraries'
+require('lib/setupTreema')
+treemaExt = require 'core/treema-ext'
+
+module.exports = class EditView extends RootView
+  id: 'editor-edit-view'
+  template: template
+
+  resource: null
+  schema: null
+  redirectPathOnSuccess: null # id or slug will be automatically added
+  filePath: null
+  resourceName: null # used in breadcrumbs
+
+  events:
+    'click #save-button': 'onClickSaveButton'
+    'click #i18n-button': 'onPopulateI18N'
+
+  constructor: (options) ->
+    super options
+
+  onLoaded: ->
+    super()
+    @buildTreema()
+    @listenTo @resource, 'change', =>
+      @resource.updateI18NCoverage()
+      @treema.set('/', @resource.attributes)
+
+  buildTreema: ->
+    return if @treema? or (not @resource.loaded)
+    data = $.extend(true, {}, @resource.attributes)
+    options =
+      data: data
+      filePath: "#{@filePath}/#{@resource.get('_id')}"
```

Comment:
This doesn't necessarily work; I notice an episode on staging has `null` in the file paths. Maybe we shuold throw an error if the subclass doesn't set a `filePath`.
<img width="621" alt="Screen Shot 2022-06-15 at 17 18 55" src="https://user-images.githubusercontent.com/99704/173963281-0d16ab6f-3c35-4740-a3dc-9728148c2262.png">